### PR TITLE
Fix copyright and author lines for iosxr lib and iosxr_template

### PR DESCRIPTION
**Pull request by rcarrillocruz**
##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME
iosxr_template

##### SUMMARY
Fix copyright and author lines for iosxr lib and iosxr_template

### CODE REVIEW

File: lib/ansible/module_utils/iosxr.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # still belong to the author of the module, and may assign their own license
 # to the complete work.
 #
-# Copyright (c) 2015 Peter Sprygada, <psprygada@ansible.com>
```
Comment (umago) [inline lib/ansible/module_utils/iosxr.py]: Shouldn't we keep this line assuming Peter is the original author of this module ? I don't think that multiple copyright lines is a problem.

Comment (rcarrillocruz) [reply lib/ansible/module_utils/iosxr.py]: yeah, just chatted with @privateip , agreeds to add a new line with RedHat, thanks

File: lib/ansible/modules/network/iosxr/_iosxr_template.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
 ---
 module: iosxr_template
 version_added: "2.1"
-author: "Peter Sprygada (@privateip)"
```
Comment (gundalow) [inline lib/ansible/modules/network/iosxr/_iosxr_template.py:27]: We support having multiple authors listed
```
author:
  - "Peter Sprygada (@privateip)"
  - "Ricardo Carrillo Cruz (@rcarrillocruz)"
```

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by rcarrillocruz** [comment]
I agreed with @gundalow to leave as is, as @privateip wants to replace the single author line.

**Pull request merged**